src/website/ccpa-modal.js

```diff
@@ -0,0 +1,14 @@
+export function CCPAModal() {
+  document.addEventListener("DOMContentLoaded", function () {
+    if(window.location.hash.substring(1) === 'your-privacy-choices') {
```

Comment:
Is this going to work when used with queryparams? jwt.io allows you to add a token with the following queryparams, `token`, `value`, `id_token` and `access_token`, which will be pasted into the debugger. eg `jwt.io?token=ey....`

Reply:
Good catch. I replaced it with `includes()` instead of `substring`. Thanks!